python/apps/utils/rabbitmq.py

```diff
@@ -84,3 +84,73 @@ def call(self, queryset=None, serializer=None, exchange_name='', exchange_type='
         print(" [.] Got %r" % self.response)
 
         return self.response
+
+    def on_response(self, ch, method, properties, body):
+        print('I have got the messsage from server')
+        self.response = body
+
+
+
+    def call_java(self, queryset=None, serializer=None, exchange_name='', exchange_type='topic', q_receiving='',
```

Comment:
Cannot see any from `Rabbitmq` class, cause every time you call the method, you are re-declaring attributes. May be you should initialize them once and reuse them :+1:

Reply:
I thought about this. Initializing them once is a good option, however there would be too many input variables while instantiating class. I decided to separate them using different methods.